protocol-units/bridge/config/src/common/movement.rs

```diff
@@ -113,15 +160,24 @@ impl MovementConfig {
 
 	pub fn suzuka() -> Self {
 		MovementConfig {
-			movement_signer_key: Ed25519PrivateKey::from_encoded_string("0x0000000000000000000000000000000000000000000000000000000000000001").unwrap(),
-			movement_native_address: "0xf90391c81027f03cdea491ed8b36ffaced26b6df208a9b569e5baf2590eb9b16".to_string(),
+			movement_signer_key: Ed25519PrivateKey::from_encoded_string(
+				"0x0000000000000000000000000000000000000000000000000000000000000001",
```

Comment:
Just want to get some implications right here. The type here should indeed be a signing key, right? Not the framework address or the genesis signer address for test (which is also not 0x1 it's 0xa12...).

Reply:
This method `suzuka()` is for testing the `bridge-service` e2e against a full running suzuka node locally. It only gets called when running the e2e test. This needs to be set to this value as this is the value of `MAPTOS_PRIVATE_KEY` when running suzuka locally. 

It may be beneficial to rename this method to `suzuka_e2e_test`.

Reply:
Yeah, agreed because it's only used in testing, renaming it `suzuka_for_testing` would be good. It's used in client tests against the framework as well.

The default signer key is `0x1` when starting a local Suzuka node. Default `config.yaml` generated is:

```
---
profiles:
  default:
    network: Custom
    private_key: "0x0000000000000000000000000000000000000000000000000000000000000001"
    public_key: "0x4cb5abf6ad79fbf5abbccafcc269d85cd2651ed4b885b5869f241aedf0a5ba29"
    account: f90391c81027f03cdea491ed8b36ffaced26b6df208a9b569e5baf2590eb9b16
    rest_url: "http://0.0.0.0:30731/"
    faucet_url: "http://0.0.0.0:30732/"
```